tools/assimp_view/Material.cpp

```diff
@@ -627,7 +627,7 @@ void CMaterialManager::HMtoNMIfNecessary(
     {
         union
         {
-            struct {unsigned char b,g,r,a;};
+            struct {unsigned char b,g,r,a;} _data;
```

Comment:
Don't start identifiers with underscore, some of those are reserved for implementation only.

Reply:
@turol Thanks. I changed it to data.